scripts/deployment/bridges/base/README.md

```diff
@@ -1,47 +0,0 @@
-# Bridge-related deployment scripts
```

Comment:
Merging Optimism and Base bridging, just leaving out their different globals and tests.